pkg/configdb/configdb.go

```diff
@@ -0,0 +1,77 @@
+package configdb
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"path"
+	"sync"
+)
+
+var mu sync.Mutex
+
+type GlobalOptions struct {
+	IP                 string
+	Gateway            string
+	LeaseDuration      int
+	AuthToken          string
+	ListenInterface    string
+	HTTPListenAddress  string
+	TLSEnabled         bool
+	TLSPrivateKeyFile  string
+	TLSCertificateFile string
+	Options            interface{}
+}
+
+type Clients struct {
+	Clients []Client
+}
+
+type Client struct {
+	Hostname string
+	MAC      string
+	IP       string
+	Options  interface{}
+}
+
+var DB *Clients
+var Config *GlobalOptions
+
+func Init(config_path string) error {
```

Comment:
Variable names usually use mixed caps as per the Effective Go book: https://go.dev/doc/effective_go#mixed-caps. I suggest taking a look at it.

Reply:
Fixed